File: JAVA/CharSort/Entity/OutCharStringGetter.java
package CharSort.Entity;

import CharSort.Interface.*;
import CharSort.*;
import java.util.*;

public class OutCharStringGetter implements IOutCharStringGetter
{
	public final String GetOutCharString(List<InChar> outChars)
	{
		char[] chrOutArray = outChars.Select(s -> s.InCharInstance).ToArray();
		return new String(chrOutArray);
	}
}

File: CharSort/Entity/OutCharStringGetter.cs
using System.Collections.Generic;
using CharSort.Interface;
using System.Linq;

namespace CharSort.Entity
{
    internal class OutCharStringGetter : IOutCharStringGetter
    {
        public string GetOutCharString(IList<InChar> outChars)
        {
            char[] chrOutArray = outChars.Select(s => s.InCharInstance).ToArray();
            return new string(chrOutArray);
        }
    }
}